src/migrations/03-create-requests.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
         key: 'id'
       }
     },
+    status: {
+      type: Sequelize.STRING,
```

Comment:
Why aren't you using an `ENUM` datatype here? @NonsoAmadi10 implemented his feature using the enum data type. You can reach out to him if you have troubles implementing it.

Reply:
I did but it throws this error `ERROR: type "public.enum_Requests_status" does not exist`
Most of all the solutions I googled suggest I downgrade my postgres from p7 to p6